README.md

```diff
@@ -73,7 +73,7 @@ production at the moment. It may be configured to do so in the future.
 Contribute
 ----------
 
-- Please read our [Contributing guidelines](CONTRIBUTING.md), [Code of Conduct](code_of_conduct.md) and [Reporting Guidelines](reporting_guidelines.md)
+- Please read our [Contributing guidelines](CONTRIBUTING.md), [Code of Conduct](code_of_conduct.md) and [Reporting Guidelines](docs/reporting_guidelines.md)
```

Comment:
```suggestion
- Please read our [Contributing guidelines](CONTRIBUTING.md), [Code of Conduct](CODE_OF_CONDUCT.md) and [Reporting Guidelines](docs/reporting_guidelines.md)
```

Reply:
@SanketDG @sanchi0204 hey I don't see a CODE_OF_CONDUCT file in docs folder and the one which is there is already in the same format unlike the suggested change which after changing will lead to an error.Also this issue was about reporting guidelines so please let me know if I should change it

Reply:
Feel free to rename it though, it's a simple `git mv` command. :slightly_smiling_face: